examples/platforms/cc2652/flash.c

```diff
@@ -28,43 +28,21 @@
 
 #include <string.h>
 
+#include <openthread/instance.h>
+
 #include <driverlib/aon_batmon.h>
 #include <driverlib/flash.h>
 #include <driverlib/interrupt.h>
 #include <driverlib/vims.h>
 
-#include <openthread-core-config.h>
-#include <openthread/config.h>
-#include <openthread/platform/alarm-milli.h>
-
 #include <utils/code_utils.h>
 
 #include "platform-cc2652.h"
 
-/*
- * The settings configuration base address *MUST* be defined in the core config
- * file. The base address *MUST* be aligned on an 8K page boundary or the flash
- * program calls will fail.
- */
-#ifndef SETTINGS_CONFIG_BASE_ADDRESS
-#error "SETTINGS_CONFIG_BASE_ADDRESS not defined in the OpenThread Core Config"
-#endif /* SETTINGS_CONFIG_BASE_ADDRESS */
-
-/*
- * The settings configuration page size *MUST* be defined in the core config
- * file. The page size *MUST* be 8K or the flash program calls will fail.
- */
-#if (SETTINGS_CONFIG_PAGE_SIZE != 0x2000)
-#error "SETTINGS_CONFIG_PAGE_SIZE must be defined in OpenThread Core Config"
-#endif
-
-/*
- * The settings configuration page number _SHOULD_ be defined in the core
- * config file.
- */
-#ifndef SETTINGS_CONFIG_PAGE_NUM
-#warn "SETTINGS_CONFIG_PAGE_NUM not defined in the OpenThread Core Config"
-#endif /* SETTINGS_CONFIG_PAGE_NUM */
+#define FLASH_BASE_ADDRESS 0x52000
+#define FLASH_PAGE_SIZE 0x2000
+#define FLASH_PAGE_NUM 2
```

Comment:
cc1352 and cc2652 function the same in this case, the comments would also apply here.